Sources/News.php

```diff
@@ -2290,7 +2298,7 @@ function getXmlPosts($xml_format, $ascending = false)
 						'content' => array(
 							array(
 								'tag' => 'name',
-								'content' => $row['poster_name'],
+								'content' => !empty($user_info['name']) ? $user_info['name'] : $user_info['username'],
```

Comment:
This will use the current user data, is this intended?  I'm thinking an scenario where an admin or any other user with the relevant permission will export someone elses data.

Reply:
Good catch. I'll add some logic to make this decision more intelligently.